general-concepts/slotting/text.xml

```diff
@@ -78,22 +175,43 @@ can then request to be automatically rebuilt when the installed version of
 <c>foo:2</c> or <c>foo:1</c> changes sub-slots <d/> for example, when the user
 upgrades from <c>foo-2.0</c> to <c>foo-2.1</c>.
 </p>
+</body>
+</subsubsection>
+
+<subsubsection>
+<title>Multiple libraries within a single package</title>
+<body>
 
 <p>
-If an ebuild does not explicitly declare a sub-slot, the regular slot is used
-as the value of the sub-slot by default.
+A package might need to install several libraries. The canonical example
+of this is <c>media-video/ffmpeg</c>:
 </p>
 
-<note>
-Care must be taken when using sub-slots in a library ebuild for the first time.
-Adding sub-slots will trigger rebuilds for all the packages that already use sub-slot
-dependencies (e.g. Switching from SLOT="0" to SLOT="0/14" in <c>media-libs/libpng</c> and
-package <c>foo</c> depends on <c>libpng:0=</c>).
-Therefore, it's best if you start using sub-slots in the library when the existing library
-interface changes.
-</note>
+<codesample lang="ebuild">
+# Subslot: libavutil major.libavcodec major.libavformat major
```

Comment:
This isn't clear on first glance, because the blank is a stronger visual separator than the dot. Include the components in pointy `<>` brackets, or write them as `libavutil_major` etc.?

Reply:
Agreed, I've actually gone for both.